Kubernetes/windows/kubeadm/KubeCluster.ps1

```diff
@@ -142,8 +142,11 @@ if ((Test-Path env:GITHUB_SDN_BRANCH) -and ($env:GITHUB_SDN_BRANCH -ne ''))
     $_GithubSDNBranch = $env:GITHUB_SDN_BRANCH
 }
 
-LoadPsm1 -Path "https://raw.githubusercontent.com/$_GithubSDNRepository/$_GithubSDNBranch/Kubernetes/windows/helper.v2.psm1"
-LoadPsm1 -Path "https://raw.githubusercontent.com/$_GithubSDNRepository/$_GithubSDNBranch/Kubernetes/windows/hns.psm1"
+LoadPsm1 -Path "c:\helper.v2.psm1"
```

Comment:
This will be reverted before merge

Reply:
Is this ready for merge?

Reply:
Let me update. I'll upload these into this repo first